label.go

```diff
@@ -23,27 +23,27 @@ func newLabels() *labels {
 
 // Label adds an optional label to the payload in the format of `key: value`.
 // Key of label must start with `labels.`
-func Label(key, value string) *label {
-	return &label{key: "labels." + key, value: value}
+func Label(key, value string) *LabelSource {
+	return &LabelSource{Key: "labels." + key, Value: value}
 }
 
 // Labels takes label structs, filters the ones that have their key start with the
```

Comment:
Please fix the comment as below

```suggestion
// Labels takes LabelSource structs, filters the ones that have their key start with the
```

Reply:
applied !!